src/components/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    // No additional reducers for categories slice
+  },
```

Comment:
It is required in this milestone that the categories slice should have a reducer that checks status and always return "under construction" kindly do well to implement this😉.

<img width="993" alt="image" src="https://github.com/riskmaris/bookstore/assets/64643995/16a51846-7809-40b9-ad71-522080710ac0">